.github/workflows/ci.yml

```
@@ -18,7 +18,7 @@ jobs:
         - name: Download dependencies
           run: |
             git clone -b foxy https://github.com/eProsima/Micro-CDR src/Micro-CDR 
-            git clone -b foxy https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client 
+            git clone -b feature/client_api_update https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client 
```

```suggestion
            git clone -b foxy https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client 
```
When https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/205 merged